### PR TITLE
docs: update to version 1 in links

### PR DESCRIPTION
**Pull request by luwes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,16 +27,14 @@ _From [Mux](https://mux.com?utm_source=github&utm_medium=social&utm_campaign=med
 
 Visit the [official documentation for Media Chrome](https://media-chrome.org/en/get-started) for the latest up-to-date usage instructions.
 
-You can find the [current release candidate documentation here.](https://v1.media-chrome.org/en/get-started)
```
Comment (gkatsev) [inline README.md:30]: should we invert this? And then in a month or two, we can remove this.

```
You can find the [documentation for v0.x of media-chrome here.](https://v0.media-chrome.org/en/get-started)
```

File: README.md [hunk 2]
```diff
@@ -27,16 +27,14 @@ _From [Mux](https://mux.com?utm_source=github&utm_medium=social&utm_campaign=med
...
-
 ## Examples
 
 <a href="https://media-chrome.mux.dev/examples/vanilla/" target="_blank">See all of the repo examples in action.</a>
 
 ### Video Example
 
 ```html
-<script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome@0.21/+esm"></script>
+<script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome@1.0/+esm"></script>
```
Comment (gkatsev) [inline README.md:37]: do we want these pointing at the major or the minor? Less risk if we pin to a minor, but then we have to be more vigilant in keeping the links up to date.

**Review by gkatsev — APPROVED** [review]
Approved, but had a couple of questions.

File: README.md (hunk 1 above)
Comment (luwes) [reply README.md:30]: makes sense, will edit

**Pull request merged**